onehungrymind/sermon-beacon
# SB-54: Fix the date picker for the sermon dialog

## Feature/Problem
The date picker in the sermon details dialog was one day behind.
When a sermon was created or deleted the table was not reflecting these results.


## Solution
Change the sermon date column type from `date` to `timestamp with timezone`
Tap the add and delete effects in sermons and load the sermon speakers.


## Areas of Impact
Sermon Table
Sermon Dialog


## UI Images



## Manual Testing
- Verify when a sermon is created or deleted the sermon table reflects that change immediately.
- Verify the date input in the sermon details within the dialog contains today's date.
- Verify CI passes
